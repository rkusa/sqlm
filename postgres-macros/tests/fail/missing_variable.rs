fn main() {
    sqlm_postgres_macros::sql_unchecked!("{id}");
}
