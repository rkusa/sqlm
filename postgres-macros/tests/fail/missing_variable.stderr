error[E0433]: failed to resolve: could not find `sql_unchecked` in `sqlm_postgres_macros`
 --> tests/fail/missing_variable.rs:2:27
  |
2 |     sqlm_postgres_macros::sql_unchecked!("{id}");
  |                           ^^^^^^^^^^^^^ could not find `sql_unchecked` in `sqlm_postgres_macros`
